app/models/student_loans/eligibility.rb

```diff
@@ -21,8 +21,8 @@ class Eligibility < ApplicationRecord
       SUBJECT_ATTRIBUTES,
     ].flatten.freeze
     ATTRIBUTE_DEPENDENCIES = {
-      "claim_school_id" => "employment_status",
-      "had_leadership_position" => "mostly_performed_leadership_duties",
+      "claim_school_id" => ["employment_status", "taught_eligible_subjects", *SUBJECT_ATTRIBUTES],
```

Comment:
👍 I wonder if we should switch the order of these so that they follow the same order as the pages in the sequence?